_configuration/tests/test_configuration.py

```diff
@@ -0,0 +1,164 @@
+# Copyright The OpenTelemetry Authors
+
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.configuration._internal.path_function import set_resource
+from opentelemetry.configuration import (
+    resolve_schema,
+    process_schema,
+    create_object,
+    validate_configuration,
+    load_configuration,
+    substitute_environment_variables,
+    render_schema,
+)
+from unittest.mock import patch
+from os import environ
+from pathlib import Path
+from pytest import fail
+
+data_path = Path(__file__).parent.joinpath("data")
+
+
+def test_create_object():
```

Comment:
Hi Diego -- not sure if I'm doing something wrong but I wanted to walk through this change and this test is failing for me. The `validate_configuration` function is looking for a file at `schema/opentelemetry_configuration.json` which doesn't exist (there's one in `_configuration/tests/data` however.

Reply:
Right, thanks for pointing that out, fixed